.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,6 +13,8 @@
 :cl:
 add: Added new things
 add: Added more things
+content: Expanded an existing thing
```

Comment:
I wonder if you're going to need to pick a different name than "content", since this is basically just a synonym for feature.

Reply:
I can do this, just got to think of an alternate name that still explains what it is